pkg/translator/cloudwatch/metadata.yaml

```
@@ -0,0 +1,3 @@
+status:
+  codeowners:
+    active: []
```

Please add yourself as a code owner here.

---

It still fails since I am not a member. Any idea of the next step?

---

It's not a required test, so I will let it fail.